ddtrace/contrib/httpx/__init__.py

```diff
@@ -57,7 +57,7 @@
 Instance Configuration
 ~~~~~~~~~~~~~~~~~~~~~~
 
-To configure particular ``httpx`` client instances use the :ref:`Pin<Pin>` API::
+To configure particular ``httpx`` client instances use the :func:`Pin<ddtrace.Pin>` API::
```

Comment:
```suggestion
To configure particular ``httpx`` client instances use the :class:`Pin<ddtrace.Pin>` API::
```

?

`:func:` seems to also work/link properly, does it matter here?